From 6aca2e444346bb31ba9225d5fa1012aa05a05a12 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 15 Jul 2024 09:44:42 +0200
Subject: [PATCH 1060/1695] dts: stm32: default enable TAMP button on
 stm32mp257f-ev1 board

Default enable TAMP button on stm32mp257f-ev1 board. The tamper is in
confirmed mode.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c4b48f7ba078983f0f0867d9897a42068d460185 dts: stm32: default enable TAMP button on stm32mp257f-ev1 board]
Change-Id: If6306bae8b9c88ea7da406b4ca8e6a08051e608d
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/393230
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 2d39a73c9..daccb4beb 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -491,7 +491,7 @@
 
 	/* Tamper button */
 	tamp-button {
-		status = "disabled";
+		status = "okay";
 		tamper-gpios = <&gpioi 8 0>;
 		st,tamp-mode = <TAMPER_CONFIRMED_MODE>;
 		st,tamp-id = <1>;
-- 
2.47.2

